### DISCUSSION TITLE
Collab?

### DISCUSSION THREAD

**Discussion by Qfc9** [zeyad-mansour/lunar#162] [Ideas]
Tried reaching out via email a month or so ago after a discord member made us aware of your project. Im the president of a nonprofit called RootKit. Our projects are pretty similar. I think a collab would be dope and we can get you some more exposure. 
https://github.com/RootKit-Org/AI-Aimbot

**Comment by toasty2436** [comment]
Hey I think I have fixed the code and redone the whole thing. I beta tested it with 3 people and they say it works very good. I am going to publicly release it at https://discord.gg/QzS9QgU7wJ. It will be free and open source!
